modules/docker/Makefile.build

```diff
@@ -0,0 +1,43 @@
+## Use DOCKER_IMAGE_NAME envvar to specify docker image with tags
+## User ARGS to pass arguments
+DOCKER_BUILD_PATH ?= .
+DOCKER_BUILD_FLAGS ?= --no-cache
+DOCKER_IMAGE_NAME ?= tests
```

Comment:
This seems like a odd default to me.

Would we want to do something like raise an error if `DOCKER_IMAGE_NAME` isn't set and this module isn't imported?

Reply:
I can take out the default. All the targets that use the image name should raise an error if it isn't set.